src/test/java/edu/cornell/kfs/module/ar/document/validation/impl/CuTotalAmountBilledToDateExceedsAwardTotalSuspensionCategoryTest.java

```diff
@@ -0,0 +1,143 @@
+        assertFalse(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByBudgetAmountEqualBudget() {
+        configureParameterServiceForSuspensionCheck(Boolean.TRUE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(50), STANDARD_AWARD_TOTAL, STANDARD_BUDGET_TOTAL);
+        assertFalse(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByBudgetAmountMoreThanBudget() {
+        configureParameterServiceForSuspensionCheck(Boolean.TRUE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(55), STANDARD_AWARD_TOTAL, STANDARD_BUDGET_TOTAL);
+        assertTrue(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByBudgetAmounNullBudgetTotalt() {
+        configureParameterServiceForSuspensionCheck(Boolean.TRUE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(1), STANDARD_AWARD_TOTAL, null);
+        assertTrue(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByAwardTotalAmountLessThanBudget() {
+        configureParameterServiceForSuspensionCheck(Boolean.FALSE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(25), STANDARD_AWARD_TOTAL, STANDARD_BUDGET_TOTAL);
+        assertFalse(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByAwardTotalAmountLessThanAwardTotal() {
+        configureParameterServiceForSuspensionCheck(Boolean.FALSE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(55), STANDARD_AWARD_TOTAL, STANDARD_BUDGET_TOTAL);
+        assertFalse(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByAwardTotalAmountEqualAwardTotal() {
+        configureParameterServiceForSuspensionCheck(Boolean.FALSE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(100), STANDARD_AWARD_TOTAL, STANDARD_BUDGET_TOTAL);
+        assertFalse(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    @Test
+    public void testSuspensionByAwardTotalAmountMoreThanAwardTotal() {
+        configureParameterServiceForSuspensionCheck(Boolean.FALSE);
+        prepareContractsGrantsInvoiceDocument(new KualiDecimal(105), STANDARD_AWARD_TOTAL, STANDARD_BUDGET_TOTAL);
+        assertTrue(suspensionCategory.shouldSuspend(contractsGrantsInvoiceDocument));
+    }
+    
+    private void configureParameterServiceForSuspensionCheck(Boolean value) {
+        Mockito.when(parameterService.getParameterValueAsBoolean(ArConstants.AR_NAMESPACE_CODE, 
+                ArConstants.CONTRACTS_GRANTS_INVOICE_COMPONENT, CuArConstants.CG_INVOICE_AMT_BILLED_SUSPENSION_BY_BUDGET_TOTAL, Boolean.TRUE.booleanValue())).thenReturn(value);
```

Comment:
Another area where Boolean unboxing may not be needed.

Reply:
OK